test/tests.js

```diff
@@ -615,6 +615,7 @@ testCase("Lame class", () => {
                     "comment": "test comment",
                     "track": "3",
                     "genre": "test genre",
+                    "artwork": "testpic.jpg",
```

Comment:
Could you please attach an image to your (e. g. 512x512 PNG with a color space) commit so that it provides an image that can be used in the tests?

Reply:
Done.